src/pymovements/datasets/sb_sat.py

```diff
@@ -99,14 +99,14 @@ class SBSAT(DatasetDefinition):
     name: str = 'SBSAT'
 
     mirrors: tuple[str, ...] = (
-        'https://files.de-1.osf.io/v1/resources/cdx69/providers/osfstorage/',
+        'https://osf.io/download/',
     )
 
     resources: tuple[dict[str, str], ...] = (
         {
-            'resource': '64525979230ea6163c031267/?zip=',
+            'resource': 'jgae7/',
             'filename': 'csvs.zip',
```

Comment:
can you also rename the filename to something like `sb-sat.csv` or so?

Reply:
changed it to: sbsat_csvs.zip